apps/system/js/notifications.js

```diff
@@ -624,18 +657,18 @@ var NotificationScreen = {
     notification.style.transform = '';
   },
 
-  addUnreadNotification: function ns_addUnreadNotification(id, skipUpdate) {
+  addUnreadNotification: function ns_addUnreadNotification(appManifestURL, skipUpdate) {
```

Comment:
I changed the signature of the function to make it self-expalantory that instead of id ,appManifestURL is being used now.